packages/neos-ui-guest-frame/src/style.css

```diff
@@ -42,7 +42,7 @@
     background-image: linear-gradient(to bottom, rgba(50, 50, 50, .1), rgba(50, 50, 50, .2), rgba(50, 50, 50, .1));
 }
 
-:global(td) {
+.ck-content .table table td {
```

Comment:
```suggestion
:global(.ck-content .table table td) {
```
But it still has to be global, right?

Reply:
Ahh, I see. Had to read docs twice to get the idea of `:global`.